install_config/persistent_storage/persistent_storage_flex_volume.adoc

```diff
@@ -0,0 +1,304 @@
+[[install-config-persistent-storage-persistent-storage-flex-volume]]
+= Persistent Storage Using Flex Volume Plugins
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+== Overview
+
+While {product-title} has built-in many
+xref:index.adoc#[volume plugins]
```

Comment:
While {product-title} has many built-in xref:../install_config/persistent_storage/index.adoc#install-config-persistent-storage-index[volume plug-ins]